include/dt-bindings/pinctrl/gr716-pinctrl.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Cobham Gaisler AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GR716_PINCTRL_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GR716_PINCTRL_H_
+
+#define GR716_IO_MODE_MAX               0x000d
+#define GR716_IO_MODE_MASK              0x000f
+
+#define GR716_IO_PULL_MASK              0x0300
```

Comment:
Please use dts/bindings/pinctrl/pincfg-node.yaml for common flags like pull up/pull down